sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/windowing/Window.java

```diff
@@ -596,6 +597,30 @@ public void validate(PCollection<T> input) {
     }
 
     @Override
+    public void populateDisplayData(DisplayData.Builder builder) {
+      builder
+          .add("windowFn", windowFn.getClass())
+          .include(windowFn)
+          .addIfNotNull("allowedLateness", allowedLateness);
+
+      if (trigger != null) {
+        builder.add("trigger", trigger.toString());
```

Comment:
the `trigger.toString()` would be evaluated before the conditional and null-ref.